COPYING

```diff
@@ -5,23 +5,42 @@ Source License, version 1.0, or at your option, any later version ("BOSL"). See
 the file ./LICENSE-BOSL for the terms of the Bootstrap Open Source Licence,
 version 1.0.
 
+In this document,
+
+- "Zcash" means the blockchain defined by the most recent agreement between
+  the Electric Coin Company and the Zcash Foundation according to section 6.2 (b)
+  of the Zcash Trademark Agreement.
+- A "Zcash Chain Fork" means a blockchain that descends from the Zcash
+  blockchain and that activates its first change to the consensus rules relative to Zcash at a block 
+  height that is or was within 3456 blocks of the current height of the Zcash
+  chain at the time of activation.
```

Comment:
```suggestion
- A "Zcash Chain Fork" means a blockchain that descends from the Zcash
  blockchain and that activates its first change to the consensus rules 
  relative to Zcash at a block height that is or was within 3456 blocks 
  of the current height of the Zcash chain at the time of activation.
```

Reply:
Is `3456` a placeholder value, or is there a justification for this specific value?